Dockerfile

```diff
@@ -17,12 +17,12 @@ RUN glide install
 
 RUN make build
 
-FROM centos:7
+FROM gcr.io/distroless/base
```

Comment:
29 CVE are present in this base image !

<img width="605" alt="Screenshot 2020-12-21 at 12 15 11" src="https://user-images.githubusercontent.com/13696927/102771513-4d825d80-4386-11eb-8c10-d0a7ef6d6a8a.png">

Reply:
Moved to `gcr.io/distroless/static` following this [documentation](https://github.com/GoogleContainerTools/distroless/tree/master/base#documentation-for-gcriodistrolessbase-and-gcriodistrolessstatic).
No vulnerabilities found with this image.

Already tested on `kube-test-eu-1`:
```log
❯ k logs http2back-oom-f5bd47c48-lbfv6 -f
2020/12/22 10:20:12 Starting http server on 0.0.0.0:8080 using provider AwsS3 (Bucket: mirakl-process-dump, Region: eu-west-1, Destination: heap_dumps/kube-test-eu-1) - Memory per buffer per file: 8 MiB
2020/12/22 10:25:27 Called : PUT / - file save-cg-pherve-test-user-5dfbd9d6f7-rd5bp-1608632726.tgz -> https://s3.eu-west-1.amazonaws.com/mirakl-process-dump/heap_dumps/kube-test-eu-1/cg-pherve-test/user/20201222_102526-098081.save-cg-pherve-test-user-5dfbd9d6f7-rd5bp-1608632726.tgz
2020/12/22 10:25:32 Called : PUT / - file cg-pherve-test-user-5dfbd9d6f7-rd5bp-1608632727.hprof -> https://s3.eu-west-1.amazonaws.com/mirakl-process-dump/heap_dumps%2Fkube-test-eu-1%2Fcg-pherve-test%2Fuser%2F20201222_102529-065466.cg-pherve-test-user-5dfbd9d6f7-rd5bp-1608632727.hprof
```